website/docs/components/alert/partials/content/content.md

```diff
@@ -0,0 +1,151 @@
+  <A.Description>Lorem ipsum dolar sit amet.</A.Description>
+  <A.Button @text="Your action" @color="critical" @onClick={{this.noop}} />
+</Hds::Alert>
+!!!
+
+### Links
+
+#### Within page and inline Alerts
+
+Use a [Standalone Link](/components/link/standalone) when an action takes the user to a new destination (URL).
+
+We recommend using the `secondary` Standalone Link to maintain visual hierarchy and avoid competing prominence when used with the `secondary` Button.
+
+<Hds::Alert @type="inline" @color="highlight" as |A|>
+  <A.Title>Links in alerts</A.Title>
+  <A.Description>Lorem ipsum dolar sit amet.</A.Description>
+  <A.Button @text="Your action" @color="secondary" @onClick={{this.noop}} />
+  <A.Link::Standalone @icon="arrow-right" @iconPosition="trailing" @color="secondary" @text="Standalone link" @href="#" />
+</Hds::Alert>
+
+If linking within the Alert description, use the `secondary` [Inline Link](/components/link/inline).
+
+<Hds::Alert @type="inline" @color="warning" as |A|>
+  <A.Title>Links in the description</A.Title>
+  <A.Description>
+    Lorem ipsum dolor sit amet, <Hds::Link::Inline @href="#" @color="secondary">consectetur</Hds::Link::Inline> adipiscing elit, sed do <Hds::Link::Inline @href="#" @color="secondary">eiusmod</Hds::Link::Inline> tempor incididunt ut <Hds::Link::Inline @href="#" @color="secondary">labore</Hds::Link::Inline> et dolore magna aliqua.
+  </A.Description>
+</Hds::Alert>
+
+If linking to multiple related items on the same page, e.g., highlighting multiple form validation errors, wrap the links in a list.
+
+<Hds::Alert @type="inline" @color="critical" as |A|>
+  <A.Title>Error submitting the form</A.Title>
+  <A.Description>
+    The follow fields must be corrected before submitting the form:
+    <li class="hds-typography-body-200">
+      <Hds::Link::Inline @color="secondary" @href="#">Email address</Hds::Link::Inline>
+    </li>
+    <li class="hds-typography-body-200">
+      <Hds::Link::Inline @color="secondary" @href="#">First name</Hds::Link::Inline>
+    </li>
+    <li class="hds-typography-body-200">
+      <Hds::Link::Inline @color="secondary" @href="#">Billing address</Hds::Link::Inline>
+    </li>
+  </A.Description>
+</Hds::Alert>
+
+#### Within compact Alerts
+
+We recommend using `secondary` [Inline Links](/components/link/inline) in the `compact` Alert to match the intended hierarchy of the component and when linking to multiple URLs.
+
+<Hds::Alert @type="compact" @color="success" as |A|>
+  <A.Description>
+    You’ve successfully configured your account. Next, <Hds::Link::Inline @href="#" @color="secondary">create a cluster</Hds::Link::Inline>, <Hds::Link::Inline @href="#" @color="secondary">invite your teammates</Hds::Link::Inline>, or create a <Hds::Link::Inline @href="#" @color="secondary">HashiCorp Virtual Network (HVN)</Hds::Link::Inline>.
+  </A.Description>
+</Hds::Alert>
+
+If the link requires more prominence, consider using the `primary` color.
```

Comment:
It's not clear if this applies also to the `Page` and `Inline` variants of the `Alert`, and –if not– if it's simply not recommended or "forbidden".

Reply:
This is directly under the headline "Within compact alerts", so I would expect it only apply to compact alerts. But maybe it can made clearer by outlining something like this:

- In a `Compact` alert: we recommend using secondary inline links within the description.
    - It's use a primary link if it requires more prominence, but don't do this if there are multiple links within the description.
 - In `Page` and `Inline` alerts: if linking within the description of the component, use secondary inline links.
    -  If the link requires more prominence, use a standalone link in the actions/footer.

Reply:
Works well now. 👍